fe/src/main/java/org/apache/doris/external/EsRestClient.java

```diff
@@ -137,10 +135,9 @@ private String execute(String path) {
             if (!(currentNode.startsWith("http://") || currentNode.startsWith("https://"))) {
                 currentNode = "http://" + currentNode;
             }
-
             Request request = builder.get()
-                    .url(currentNode + "/" + path)
-                    .build();
+                .url(currentNode + "/" + path)
+                .build();
             LOG.trace("es rest client request URL: {}", currentNode + "/" + path);
```

Comment:
```suggestion
          if (LOG.isTraceEnabled) {
            LOG.trace("es rest client request URL: {}", currentNode + "/" + path);
            }
```

Reply:
Ok, I will modify it，I forgot to judge it here